pvlib/iotools/maccrad.py

```diff
@@ -0,0 +1,155 @@
+# standard library imports 
+import logging
+
+# related third party imports
+import pandas as pd
+import pytz
+
+# local application/library specific imports 
+from pvlib.location import Location
+from pvlib.iotools.util import get_loc_latlon, localise_df
+
+# required dateconverters
+def dtp_soda_pro_macc_rad(date):
+    """
+    datetime converter for MACC-RAD data and others
+    """
+    datetime_stamp = date.split('/')[0]
+
+    return datetime_stamp
+    
+#XXX read metadata / header
+
+def read_maccrad_metadata(file_csv, name='maccrad'):
+    """
+    Read metadata from commented lines of the file
+    * coordinates
+    * altitude
+    
+    Retrieve timezone
+    """
+    if not name:
+        name = file_csv.split('.')[0]
+    
+    ## if file is on local drive
+    f = open(file_csv)
+    for line in f:
+        if "Title" in line:
+            title = line.split(':')[1].split('(')[0].strip()
+            name = title
```

Comment:
I'm confused about what you want ``name`` to be set to. The title line contains a generic description of the data source.